ios/engine/KMEI/KeymanEngine/resources/Keyman.bundle/Contents/Resources/keyboard.html

```diff
@@ -272,8 +272,13 @@
                 if(undefined == text) text = '';
                 var ta = document.getElementById('ta');
                 var kmw = window['keyman'];
+
+                let resetContext = ta.value != text
```

Comment:
```suggestion
                var resetContext = ta.value != text;
```

(we aren't using `let` elsewhere in keyboard.html, and should include the semicolon)

Reply:
The dangers of language-hopping.  It's hard to keep the syntax straight sometimes.